{% extends "layout.html" %}
{% block title %}Model{% endblock %}

{% block page_content %}

<section class="header-box"></section>

<div class="px-2 px-lg-5 d-flex flex-row">
  <div class="d-none d-lg-block" style="min-width: 280px; position: relative">
    <div id="anchors">
      <div class="title">Contents</div>
      <ul id="anchors-list"></ul>
    </div>
  </div>
  <section class="py-5 container wiki-editor-container"><p style="text-align: justify;"><em>Spodoptera litura </em>(common cutworm) is a polyphagous agricultural pest worldwide and causes enormous losses to a myriad of economical crops. It has evolved high resistance to multiple chemical insecticides. It poses a necessity for development of alternative strategies for S. litura control. Recently, RNA interference (RNAi) technology has emerging as a promising strategy for pest insect management. In this project, we attempted to express artificial microRNA (amiRNA) targeting the chitin synthase 1 (<em>SlCHS1</em>) gene of <em>S. litura</em> from the genome of tobacco plastid. To protect the degradation of amiRNA caused by highly active nuclease in <em>S. litura</em>, amiRNA was delivered by a bacteriophage MS2 virus-like particle (VLP)-based delivery system. To better understand the plastid-mediated RNAi approach for <em>S. litura</em> management, we established models of S. litura distribution in China and predicted the structure of CHS1.<br> </p><h2 style="text-align: justify;">1 Distribution of <em>Spodoptera litura</em> <br></h2><p><strong>1.1 Current distribution of </strong><em><strong>Spodoptera litura</strong></em><br><br>An important omnivorous or polyphagous pest found all throughout the world &nbsp;is Spodoptera litura. &nbsp;The subtropical and temperate regions of Asia and Oceania are the primary distribution areas for this insect. &nbsp;It is harmful throughout the year. &nbsp;India, Pakistan, Bangladesh, South Korea, Japan, Thailand, Philippines, Australia, New Zealand, and the United States (only Hawaii) are among the nations that are particularly impacted. &nbsp;The current global distribution map of Spodoptera litura (PRODLI) [World distribution] | EPPO Global Database is shown below.<br><img src="https://static.igem.wiki/teams/5044/model01.png" alt="..." data-href="" style=""><br> In China, S. litura is primarily found in the Yangtze River Basin, South and Southwest China, with less distribution in North, Northeast, and Northwest China.Next, <span style="color: rgb(19, 82, 0); font-size: 16px;"><strong>PLASTID </strong></span><span style="color: rgb(19, 82, 0); font-size: 16px;"> </span><span style="color: rgb(255, 255, 255); background-color: rgb(207, 19, 34); font-size: 16px;">PESTI</span><span style="color: rgb(255, 255, 255); background-color: rgb(212, 56, 13); font-size: 16px;">CIDES</span><span style="color: rgb(44, 44, 54); font-size: 16px;"><em>™</em></span> shall mostly address Spodoptera litura's distribution within China. &nbsp;</p><p><strong>1.2 Simulation prediction of potential distribution areas of </strong><em><strong>Spodoptera litura</strong></em><strong>.</strong></p><p style="text-align: justify;">According to recent research, the most significant climatic element &nbsp;influencing Spodoptera litura distribution is the lowest temperature during the coldest month.Temperature is a key environmental component that influences how insects develop. As a result of present global warming, pests will appear earlier, multiply in quantity and generation in the field, and prolong the damage period, all of which will pose a severe danger to food security.It is essential to implement targeted preventive and control measures, guarantee the safety of food production, and anticipate how agricultural pests will react to global warming and changes in their distribution areas in order to deal with the impact of climate change on agricultural pests and illnesses.Consequently, <span style="color: rgb(19, 82, 0); font-size: 16px;"><strong>PLASTID </strong></span><span style="color: rgb(19, 82, 0); font-size: 16px;"> </span><span style="color: rgb(255, 255, 255); background-color: rgb(207, 19, 34); font-size: 16px;">PESTI</span><span style="color: rgb(255, 255, 255); background-color: rgb(212, 56, 13); font-size: 16px;">CIDES</span><span style="color: rgb(44, 44, 54); font-size: 16px;"><em>™</em></span> intend to carry out study on Spodoptera litura in China. </p><p style="text-align: justify;">Modeling predictions of potential distribution areas.In species distribution modeling, the goal of the MaxEnt model is to predict a species’ potential distribution area using known species distribution data (e.g., species’presence records) and environmental variables (e.g., climate, land use, etc.).The MaxEnt model uses the maximum entropy principle to generate the least biased distribution predictions, provided that the model satisfies known constraints.</p><p style="text-align: justify;">The model form of MaxEnt is:<br><img src="https://static.igem.wiki/teams/5044/model02.png" alt="..." data-href="" style="width: 295.40px;height: 59.70px;"></p><p>Among these, <br>P(y|x) is the probability of species existence given environmental conditions<br>fi(x) is a characteristic function related to environmental conditions, indicating how environmental conditions affect the distribution of species. &nbsp;<br>λi is the weight of the characteristic function<br>Z(x) is the normalization factor.<br></p><p><strong>Dataset preparation</strong></p><p>The<em> S. litura</em> distribution data set uses part of the <em>S. litura</em> distribution data in mainland China from https://www.gbif.org/. The climate time series data uses the data from https://www.worldclim.org/ Historical climate data.Here, climate data uses three indicators: maximum &nbsp;temperature, minimum temperature, and average temperature.Import the corresponding data set into ArcMap software, and the visualization results are as follows:<br></p><p><img src="https://static.igem.wiki/teams/5044/model03.png" alt="" data-href="" style="width: 100%;"></p><p style="text-align: justify;"><strong>Figure 2: Training set sampling points. </strong></p><p style="text-align: justify;"><br>Since the historical meteorological data is global data, it is necessary to cut out China’s areas on the climate map.By using the Extract By Mask function in SDM TOOLBOX, import China’s map vector information to complete the cutting.</p><p><br><strong>The use of Maxent software </strong><br><br>The produced Spodoptera litura distribution data should be imported into Maxent software, along with climatic data. Epochs should be set to 500, and cross-validation should be used for the machine learning evaluation method.Following testing, the training set and testing were carried out &nbsp; &nbsp; &nbsp;following eighteen ground iterations.The set's AUC is 0.98, indicating a higher degree of accuracy in the results prediction.</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/model04.png" alt="..." data-href="" style="width: 50%;"></p><p style="text-align: center;"><strong>Figure 3. AUC values of the training set and test set after 18 rounds of iterations.</strong></p><p style="text-align: center;"><br>The anticipated area and miss rate as a function of cumulative threshold are displayed in the graph below. &nbsp;Training presence records and test records—if test data is used—are the basis for calculating the missing rate.The omission rate ought to be near the anticipated omissions because of how the cumulative threshold is defined.</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/model05.png" alt="..." data-href="" style="width: 50%;"></p><p style="text-align: center;"><strong>Figure 4. Missing rate and prediction area as a function of cumulative threshold.</strong></p><p style="text-align: justify;"><br>Some common thresholds and corresponding omission rates are as follows. If test data are available, binomial probabilities are calculated exactly if the number of test samples is at most 25, otherwise using a normal approximation to the binomial. These are 1-sided p-values for the null hypothesis that test points are predicted no better than by a random prediction with the same fractional predicted area. The "Balance" threshold minimizes 6 * training omission rate + .04 * cumulative threshold + 1.6 * fractional predicted area.</p><p style="text-align: justify;"><strong>Table 1. Common thresholds and corresponding miss rates.</strong></p><table style="width: auto;"><tbody><tr><th colspan="1" rowspan="1" width="132">Cumulative<br>threshold<br></th><th colspan="1" rowspan="1" width="117">Cloglog<br>threshold<br></th><th colspan="1" rowspan="1" width="307" style="text-align: left;">Description</th><th colspan="1" rowspan="1" width="118.09" style="text-align: left;">Fractional<br>predicted area	</th><th colspan="1" rowspan="1" width="111.17" style="text-align: left;">Training omission<br> rate	</th><th colspan="1" rowspan="1" width="110.96" style="text-align: left;">Test omission rate</th><th colspan="1" rowspan="1" width="177.37" style="text-align: left;">P-value</th></tr><tr><td colspan="1" rowspan="1" width="auto">1</td><td colspan="1" rowspan="1" width="auto">0.009</td><td colspan="1" rowspan="1" width="auto">Fixed cumulative value 1</td><td colspan="1" rowspan="1" width="auto">0.241</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">2.41E-01</td></tr><tr><td colspan="1" rowspan="1" width="auto">5</td><td colspan="1" rowspan="1" width="auto">0.064</td><td colspan="1" rowspan="1" width="auto">Fixed cumulative value 5</td><td colspan="1" rowspan="1" width="auto">0.132</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">1.33E-01</td></tr><tr><td colspan="1" rowspan="1" width="auto">10</td><td colspan="1" rowspan="1" width="auto">0.116</td><td colspan="1" rowspan="1" width="auto">Fixed cumulative value 10</td><td colspan="1" rowspan="1" width="auto">0.089</td><td colspan="1" rowspan="1" width="auto">0.056</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">8.94E-02</td></tr><tr><td colspan="1" rowspan="1" width="auto">5.011</td><td colspan="1" rowspan="1" width="auto">0.064</td><td colspan="1" rowspan="1" width="auto">Minimum training presence</td><td colspan="1" rowspan="1" width="auto">0.132</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">1.32E-01</td></tr><tr><td colspan="1" rowspan="1" width="auto">16.227</td><td colspan="1" rowspan="1" width="auto">0.206</td><td colspan="1" rowspan="1" width="auto">10 percentile training presence</td><td colspan="1" rowspan="1" width="auto">0.061</td><td colspan="1" rowspan="1" width="auto">0.056</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">6.10E-02</td></tr><tr><td colspan="1" rowspan="1" width="auto">16.227</td><td colspan="1" rowspan="1" width="auto">0.206</td><td colspan="1" rowspan="1" width="auto">Equal test sensitivity and specificity</td><td colspan="1" rowspan="1" width="auto">0.061</td><td colspan="1" rowspan="1" width="auto">0.056</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">6.10E-02</td></tr><tr><td colspan="1" rowspan="1" width="auto">16.227</td><td colspan="1" rowspan="1" width="auto">0.206</td><td colspan="1" rowspan="1" width="auto">Maximum training sensitivity plus specificity</td><td colspan="1" rowspan="1" width="auto">0.061</td><td colspan="1" rowspan="1" width="auto">0.056</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">6.10E-02</td></tr><tr><td colspan="1" rowspan="1" width="auto">41.701</td><td colspan="1" rowspan="1" width="auto">0.6</td><td colspan="1" rowspan="1" width="auto">Equal test sensitivity and specificity</td><td colspan="1" rowspan="1" width="auto">0.02</td><td colspan="1" rowspan="1" width="auto">0.0278</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">2.00E-02</td></tr><tr><td colspan="1" rowspan="1" width="auto">41.701</td><td colspan="1" rowspan="1" width="auto">0.6</td><td colspan="1" rowspan="1" width="auto">Maximum test sensitivity plus specificity</td><td colspan="1" rowspan="1" width="auto">0.02</td><td colspan="1" rowspan="1" width="auto">0.0278</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">2.00E-02</td></tr><tr><td colspan="1" rowspan="1" width="auto">2.475</td><td colspan="1" rowspan="1" width="auto">0.03</td><td colspan="1" rowspan="1" width="auto">Balance training omission, predicted area and threshold value</td><td colspan="1" rowspan="1" width="auto">0.173</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto"><br>1.73E-01</td></tr><tr><td colspan="1" rowspan="1" width="auto">12.139</td><td colspan="1" rowspan="1" width="auto">0.146</td><td colspan="1" rowspan="1" width="auto">Equate entropy of threshold and original distributions</td><td colspan="1" rowspan="1" width="auto">0.077</td><td colspan="1" rowspan="1" width="auto">0.056</td><td colspan="1" rowspan="1" width="auto">0</td><td colspan="1" rowspan="1" width="auto">7.75E-02</td></tr></tbody></table><p> </p><h4>Analysis and conclusion of the results.</h4><p> &nbsp; &nbsp;The figure below displays the forecast results following the eighteenth iteration.</p><p><img src="https://static.igem.wiki/teams/5044/model07.png" alt="..." data-href="" style="width: 100%;"></p><p> &nbsp; &nbsp; <strong>Figure 5. &nbsp;Predicted structure distribution map of </strong><em><strong>S. litura</strong></em><strong>.</strong></p><p> The purple sample denotes the testing position, and the white square indicates the sample training position. The warmer the color value, the more suitable for the survival of the species. &nbsp;Through observation, it can be seen that the color around some observation points in the eastern and southern coastal areas is close to 1, and in the &nbsp;central and southeastern areas, the color value is close to 1. The color in coastal areas is generally higher than 0.6. These findings suggest that<em> S. litura</em> has an obvious tendency to spread to the surrounding areas.<br></p><h2><br>2. Model construction in the RNAi process.</h2><p> &nbsp; &nbsp; &nbsp; &nbsp;</p><h3>2.1 Assembling AGO and siRNA into a RISC structure.</h3><p><br> &nbsp; &nbsp; &nbsp;The specific principle is as follows. In transgenic tobacco, there exists long dsRNA. After being ingested by Spodoptera litura and entering the &nbsp; &nbsp; &nbsp;target cells, the long dsRNA is processed by the RNase-III enzyme cutter &nbsp; &nbsp; &nbsp;in the target cells into small interfering RNA (siRNA) with a length of &nbsp; &nbsp; &nbsp;20-25 nucleotides. Subsequently, the Argonaute protein (AGO) assembles &nbsp; &nbsp; &nbsp;these siRNAs to form an RNA-induced silencing complex (RISC), which<br> &nbsp; &nbsp; &nbsp;mediates the degradation of endogenous mRNA that is complementary to its &nbsp; &nbsp; guide strand, thus achieving the purpose of pest control. &nbsp; &nbsp;</p><p> &nbsp; &nbsp; &nbsp;Here, <span style="color: rgb(19, 82, 0); font-size: 16px;"><strong>PLASTID </strong></span><span style="color: rgb(19, 82, 0); font-size: 16px;"> </span><span style="color: rgb(255, 255, 255); background-color: rgb(207, 19, 34); font-size: 16px;">PESTI</span><span style="color: rgb(255, 255, 255); background-color: rgb(212, 56, 13); font-size: 16px;">CIDES</span><span style="color: rgb(44, 44, 54); font-size: 16px;"><em>™</em></span> use AlphaFold3 to conduct predictive modeling of the RNA-induced &nbsp; &nbsp;silencing complex (RISC) formed by the assembly of siRNAs by the Argonaute &nbsp;protein (AGO), and the result is consistent with that of Dinshaw J. Patel &nbsp; &nbsp; &nbsp;(2014).<br> &nbsp; &nbsp; &nbsp;<img src="https://static.igem.wiki/teams/5044/model09.png" alt="" data-href="" style=""></p><h3>2.1 Assembling AGO and siRNA into a RISC structure.</h3><p style="text-indent: 24pt;">The specific principle is as follows. In transgenic tobacco, there exists long dsRNA. After being ingested by Spodoptera litura and entering the target cells, the long dsRNA is processed by the RNase-III enzyme cutter in the target cells into small interfering RNA (siRNA) with a length of 20-25 nucleotides. Subsequently, the Argonaute protein (AGO) assembles these siRNAs to form an RNA-induced silencing complex (RISC), which mediates the degradation of endogenous mRNA that is complementary to its guide strand, thus achieving the purpose of pest control.</p><p style="text-indent: 21pt;">Here, <span style="color: rgb(19, 82, 0); font-size: 16px;"><strong>PLASTID </strong></span><span style="color: rgb(19, 82, 0); font-size: 16px;"> </span><span style="color: rgb(255, 255, 255); background-color: rgb(207, 19, 34); font-size: 16px;">PESTI</span><span style="color: rgb(255, 255, 255); background-color: rgb(212, 56, 13); font-size: 16px;">CIDES</span><span style="color: rgb(44, 44, 54); font-size: 16px;"><em>™</em></span> use AlphaFold3 to conduct predictive modeling of the RNA-induced silencing complex (RISC) formed by the assembly of siRNAs by the Argonaute protein (AGO), and the result is consistent with that of Dinshaw J. Patel (2014). </p><p style="text-indent: 21pt; text-align: center;"><img src="https://static.igem.wiki/teams/5044/1.jpg" alt="图片1.jpg" data-href="" style=""></p><p style="text-indent: 21pt;"><br></p><p style="text-align: center;"><strong>Figure 6: Crystal structure of Aa-Ago with externally bound siRNA</strong>.</p><p><strong>2.2 SlCHS1 protein structure prediction.</strong></p><p style="text-indent: 24pt;">In the prevention and control of Spodoptera litura, RISC silences the SICHS gene and inhibits the synthesis of SlCHS1, so that the larvae cannot develop into pupae normally and the adults cannot completely molt, thereby preventing and controlling Spodoptera litura. &nbsp;</p><p style="text-indent: 24pt;">The cDNA sequence of SlCHS1 (GenBank accession number: high.[5] </p><p style="text-align: center;">We used swiss-model (https://swissmodel.expasy.org) to predict the protein structure of SICHS1 (Figure 7), and the Global Model Quality Estimate was 0.73. <img src="https://static.igem.wiki/teams/5044/2.jpg" alt="图片2.jpg" data-href="" style=""></p><p style="text-align: center;"><strong>Figure 7. Predicted structure of </strong><em><strong>SI</strong></em><strong>CHS1</strong></p><p style="text-indent: 24pt;">To determine the contours of favoured regions, data was extracted from 12,521 non redundant experimental structures (pairwise sequence identity cutoff 30%, X-ray resolution cutoff 2.5Å) as culled from PISCES. Histograms with a binning of 4 degrees were then used to count Φ (Phi; C-N-CA-C) / Ψ (Psi; N-CA-C-N) occurences for all displayed categories. The number of observed Φ / Ψ pairs determines the contour lines</p><p style="text-indent: 24pt;"><br></p><p style="text-indent: 24pt;"><br></p><p style="text-indent: 84.35pt; text-align: left;"><strong>Figure 8 Ramachandran Plots. </strong><span style="font-family: 宋体;">（</span>a<span style="font-family: 宋体;">）</span>General (No Proline or Glycine) &nbsp; &nbsp; &nbsp; &nbsp;<span style="font-family: 宋体;">（</span>b<span style="font-family: 宋体;">）</span>Glycine Only<span style="font-family: 宋体;">（</span>c<span style="font-family: 宋体;">）</span>Proline Only &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<span style="font-family: 宋体;">（</span>d<span style="font-family: 宋体;">）</span>Pre-Proline Only</p><p style="text-indent: 24pt;"><br></p><p><strong>References</strong><br></p><p>Qin Houguo, Wang Dudong, Ding Jian, et al. &nbsp;List of host plants of <em>Spodoptera litura</em>. &nbsp;Journal of Jiangxi Agricultural Sciences, 2006, (05): 51-58. &nbsp;DOI: 10.19386/j.cnki.jxnyxb.2006.05 .017<span style="font-family: 黑体;">．</span> </p><p>BAJPAI NK, BALLAL CP, RAO NS, et al. &nbsp;Competitive interaction be tween two ichneumonid parasitoids of Spodoptera litura<span style="font-family: 黑体;">［</span>J<span style="font-family: 黑体;">］．</span> Bio Control, 2006, 51:419-438<span style="font-family: 黑体;">．</span> </p><p>Zhang Wenxiu, Zhou Qinghua, Yang Liu, Wang Dongqi, Wu Pinghua, Qin Guangming, Cai Gaolei, Fu Hongyang, Zhou Yang. &nbsp;Simulation and prediction of the impact of climate change on the potential geographical distribution areas and overwintering boundaries of Spodoptera litura [J]. &nbsp;China Plant Protection Guide, 2023, 43(2):31-39. &nbsp;</p><p>Yuan YR, Pei Y, Chen HY, Tuschl T, Patel DJ. &nbsp;A potential protein-RNA recognition event along the RISC-loading pathway from the structure of A. &nbsp;aeolicus Argonaute with externally bound siRNA. &nbsp;Structure. &nbsp;2006 Oct;14(10):1557-65. &nbsp;doi: 10.1016/j.str.2006.08.009. &nbsp;PMID: 17027504; PMCID: PMC4692372.</p><p>Yu, H.-Z.; Li, N.-Y.; Xie, Y.-X.; Zhang, Q.; Wang, Y.; Lu, Z.-J. Identification and Functional Analysis of Two Chitin Synthase Genes in the Common Cutworm, Spodoptera litura. Insects 2020, 11, 253. </p><p><br></p><p><br></p><p><br></p><p><br></p><p><br></p><p><br></p><p><br></p><p><br></p><p><br></p><p><br></p></section>
  <div id="entry-wrapper" class="d-none d-md-block">
    <div id="entry">
      <div class="item">
        <div class="title" style="color: rgb(0, 0, 0)">the MaxEnt model</div>
        <div class="content">
          <p>
            Maxent software can be used to model species niches and
            distributions by applying a machine-learning technique called
            maximum entropy modeling. From a set of environmental (e.g.,
            climatic) grids and georeferenced occurrence localities, the model
            expresses a probability distribution where each grid cell has a
            predicted suitability of conditions for the species. Under
            particular assumptions about the input data and biological sampling
            efforts that led to occurrence records, the output can be
            interpreted as predicted probability of presence (cloglog
            transform), or as predicted local abundance (raw exponential
            output).<img src="https://onlinelibrary.wiley.com/cms/asset/21dd838b-a51e-4f3f-865f-8b2cfe579db8/ddi_725_f3.gif" alt="" data-href="" style="width: 100%">
          </p>
        </div>
      </div>
      <div class="item">
        <div class="title" style="color: rgb(0, 0, 0)">ArcMap</div>
        <div class="content">
          <p>
            <span style="color: rgb(77, 77, 77)">ArcMap is where you display and explore GIS datasets for your
              study area, where you assign symbols, and where you create map
              layouts for printing or publication.ArcMap represents geographic
              information as a collection of layers and other elements in a map.
              Common map elements include the data frame containing map layers
              for a given extent plus a scale bar, north arrow, title,
              descriptive text, a symbol legend, and so on.</span>
          </p>
        </div>
      </div>
      <div class="item">
        <div class="title" style="color: rgb(0, 0, 0)">Extract by Mask</div>
        <div class="content">
          <p>
            <span style="color: rgb(24, 44, 66)">Extract by Mask using ArcGIS It is possible to select a specific
              area of a raster using another layer (raster or entity) as a
              template whose extension delimits the extent of the output raster.
              The output raster will maintain its attribute table, bounded to
              the extension that <span style="color: rgb(19, 82, 0); font-size: 16px;"><strong>PLASTID </strong></span><span style="color: rgb(19, 82, 0); font-size: 16px;"> </span><span style="color: rgb(255, 255, 255); background-color: rgb(207, 19, 34); font-size: 16px;">PESTI</span><span style="color: rgb(255, 255, 255); background-color: rgb(212, 56, 13); font-size: 16px;">CIDES</span><span style="color: rgb(44, 44, 54); font-size: 16px;"><em>™</em></span> have imposed.</span><img src="https://www.giscourse.com/wp-content/uploads/2017/10/arcgis_mask_1.png" alt="" data-href="" style="width: 100%">
          </p>
        </div>
      </div>
      <div class="item">
        <div class="title" style="color: rgb(0, 0, 0)">SDM TOOLBOX</div>
        <div class="content">
          <p>
            <span style="color: rgb(68, 68, 68)">SDM toolbox is a python-based ArcGIS toolbox for spatial studies
              of ecology, evolution and genetics. SDMtoolbox consists of a
              series python scripts (92 and growing) designed to automate
              complicated ArcMap (ESRI) analyses. A large set of the tools were
              created to complement MaxEnt species distribution models (SDMs) or
              to improve the predictive performance of MaxEnt models (for an
              overview, see chapter 5 in the user guide Running a SDM in MaxEnt:
              from Start to Finish). MaxEnt uses maximum entropy to model
              species’ geographic distributions using presence-only data
              (Phillips et al. 2006) and has become one of the most prevalent
              methods due to its high predictive performance, computational
              efficiency and ease of use. SDMtoolbox is not limited to analyses
              of MaxEnt models and many tools are also available for use on
              other data (i.e. haplotype networks) or the results of other SDM
              methods.</span><img src="http://sdmtoolbox.org/wp-content/uploads/2013/07/overprediction_cont_BD.png" alt="" data-href="" style="width: 100%">
          </p>
        </div>
      </div>
      <div class="item">
        <div class="title" style="color: rgb(0, 0, 0)">AUC</div>
        <div class="content">
          <p>
            <span style="color: rgb(0, 0, 0)">AUC, short for area under the ROC (receiver operating
              characteristic) curve, is a relatively straightforward metric that
              is useful across a range of use-cases. In this blog, <span style="color: rgb(19, 82, 0); font-size: 16px;"><strong>PLASTID </strong></span><span style="color: rgb(19, 82, 0); font-size: 16px;"> </span><span style="color: rgb(255, 255, 255); background-color: rgb(207, 19, 34); font-size: 16px;">PESTI</span><span style="color: rgb(255, 255, 255); background-color: rgb(212, 56, 13); font-size: 16px;">CIDES</span><span style="color: rgb(44, 44, 54); font-size: 16px;"><em>™</em></span> present an
              intuitive way of understanding how AUC is calculated.</span>
          </p>
        </div>
      </div>
    </div>
  </div>
</div>

<script src="{{ url_for('static', filename = 'js/sidebar.js') }}"></script>
<script>
  initAnchors(100, 560);
  initEntry();
</script>
{% endblock %}
