{% extends "layout.html" %}
{% block title %}Results{% endblock %}

{% block page_content %}
  <section class="position-relative">
    <img src="https://static.igem.wiki/teams/5044/header-result.jpg" class="img-fluid" alt="">
  </section>
  <div class="px-2 px-lg-5 d-flex flex-row">
    <div class="d-none d-lg-block" style="min-width: 280px; position: relative">
      <div id="anchors">
        <div class="title">Contents</div>
        <ul id="anchors-list"></ul>
      </div>
    </div>
    <section class="py-5 container wiki-editor-container"><h2><span style="color: rgb(89, 191, 192);"><strong>Engineered bacteria construction</strong></span></h2><h3><span style="color: rgb(114, 192, 64);"><strong>Prediction of Effective Fragments of CHS1</strong></span></h3><p style="text-align: justify;">The deduced amino acid sequences of <em>SlCHS1</em> and <em>SlCHS2 </em>were analyzed by using DASTAR software. The open reading frame (ORF) was predicted according to the ORF finder tool (<a href="https://www.ncbi.nlm.nih.gov/orffinfer/" target="_blank">https://www.ncbi.nlm.nih.gov/orffinfer/</a>). The molecular weight (MW) and isogenic point (pI) of<em> SlCHS1</em> and <em>SlCHS2</em> were calculated using ExPASy (http://web.expasy.org/compute_pi). The signal peptides of <em>SlCHS1 </em>and <em>SlCHS2 </em>were predicted using SignalP 4.1 Server (http://www.cbs.dtu.dk/services/SignalP). The functional domain was predicted by using SMART software (http://smart.embl-heidelberg.de/). The membrane-spanning domain was predicted by TMHMM Server v. 2.0 (http://www.cbs.dtu.dk/services/TMHMM/). The phylogenetic tree was constructed with MEGA 7.0 software using the neighbor-joining method with 1000-fold bootstrap resampling. Protein sequences from different insect species were obtained from GenBank (http://www.ncbi.nlm.nih.gov/) and used in the phylogenetic analysis. </p><p style="text-align: justify;"><img src="https://static.igem.wiki/teams/5044/result-16.jpg" alt="" data-href="" style="width: 100%;"><strong>Figure 1 &nbsp;</strong>Multiple sequence alignment of the conserved catalytic domain of the chitin synthases (CHSs) from three insect species. CHSs are from <em>Spodoptera litura</em> (<em>Sl</em>), <em>Spodoptera exigua</em> (<em>Se</em>) and <em>Helicoverpa armigera</em> (<em>Ha</em>). Seven characteristic motifs (M 1–7) for insect chitin synthases are indicated with red boxes.</p><p style="text-align: center;"><br></p><h3><span style="color: rgb(114, 192, 64);"><strong>Construction of bacteria expression vectors</strong></span></h3><p style="text-align: justify;">The bacteria expression vectors were constructed based on pET28a. Three vectors were designed for expressing amiR-CHS1, VLP ( HIV TAT-conjugated coat protein (CP) dimer), and VLP plus amiR-CHS1 ( VLP-amiR-CHS1, Figure 3) in <em>Escherichia coli</em>. </p><p style="text-align: left;"><img src="https://static.igem.wiki/teams/5044/20240930012352.png" alt="微信图片_20240930012352.png" data-href="" style=""><strong>Figure 2 &nbsp;</strong>Vector design for expressing<strong> </strong>amiRCHS1, VLP and VLP-amiRCHS1 in bacteria. pac: the site where CP recognizes RNA and assembles it; TAT, transmembrane peptide. The above vectors are all modified from pET28a.</p><p style="text-align: center;"><br></p><h3><span style="color: rgb(114, 192, 64);"><strong>Detection of RNA Expression</strong></span></h3><p style="text-align: justify;">Three bacteria expression vectors were expressed in <em>E.coli </em>BL21(DE3) that lacks RNase III activity (BL21Δrnc). RNA in different strains before and after IPTG induction (overnight) was extracted and detected with formaldehyde denaturing gel. Northern blot showed the expression amiR-CHS1 in BL21Δrnc that expresses amiR-CHS1 and VLP-amiR-CHS1.</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/20240929235243.png" alt="微信图片_20240929235243.png" data-href="" style="width: 60%;"></p><p style="text-align: left;"><strong>Figure 3 &nbsp;</strong>Detection of amiRNA accumulation in BL21ΔRNC. After IPTG induction, amiRNA expression was detected in the strains expressing VLP, amiR-CHS1 and VLP-amiR-CHS1 (138 nt). &nbsp;Samples of 10 µg total cellular RNA were loaded in each lane of the RNA blot. The Gelview-stained gel before blotting is shown below the blot as a loading control.</p><p style="text-align: left;"><br></p><h3><span style="color: rgb(114, 192, 64);"><strong>Detection of Protein Expression</strong></span></h3><p style="text-align: justify;">Proteins were extracted from different BL21Δrnc strains before and after IPTG induction (overnight). Successful expression of VLP was determined by SDS-PAGE in bacteria expressing VLP and VLP-amiR-CHS1.</p><p style="text-align: justify;">In summery, both protein and amiRNA could efficiently express in BL21Δrnc.</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/20240930000118.png" alt="微信图片_20240930000118.png" data-href="" style=""></p><p style="text-align: left;"><strong>Figure 4 </strong> Detection of protein expression level in BL21ΔRNC. After IPTG induction, the expression of 2×CP (28 kDa) was detected in the VLP and VLP-amiR-CHS1-expressing strains. Sample load, 10<span style="font-family: Arial;"> μg.</span></p><p style="text-align: left;"><br></p><h2><span style="color: rgb(89, 191, 192);"><strong>Insect feeding and bioassays</strong></span></h2><h3><span style="color: rgb(114, 192, 64);"><strong>VLP improved RNAi efficiency in </strong></span><span style="color: rgb(114, 192, 64);"><em><strong>S. litura </strong></em></span></h3><p>Bioassay was conducted via feeding <em>S. litura</em> larvae with an artiﬁcial diet supplemented with the engineered bacteria. Only the VLP-amiR-CHS1 markerdly reduced the larval weight gain7 days after feeding (DAF).</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/20241001172757.png" alt="" data-href="" style=""></p><p style="text-align: left;"><strong>Figure 5 </strong> Bioassay of<em> S. litura</em> larvae 7 days after feeding (DAF). &nbsp;(A) Representative examples of <em>S. litura</em> larvae. (B) Larval weight of<em> S. litura</em>. Data are means ± SE (n ≥ 30).</p><p style="text-align: center;"><br></p><h3><span style="color: rgb(114, 192, 64);"><strong>Relative expression of </strong></span><span style="color: rgb(114, 192, 64);"><em><strong>SlCHS1</strong></em></span></h3><p style="text-align: justify;">As expected, the expression of target gene was significantly reduced in larvae fed with VLP-amiR-CHS1-expressed bacteria compared with other groups.</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/20241001173945.png" alt="" data-href="" style="width: 50%;"></p><p style="text-align: left;"><strong>Figure 6 </strong>Analysis of <em>CHS1</em> expression in <em>S. litura</em> after 7 d of feeding by qRT-PCR. Data are means ± SEM (n ≥6). Different lowercase letters above the columns indicate significant differences (<em>P</em> &lt; 0.05, one-way ANOVA with Tukey’s multiple range test). </p><p style="text-align: center;"><br></p><h3><span style="color: rgb(114, 192, 64);"><strong>Detection the stability of the amiRNA</strong></span></h3><p style="text-align: justify;">To test whether the MS2 VLP could reduce the degradation of amiRNA by nuclerase in the midgut of <em>S. litura</em>, RNA from different BL21Δrnc strains was extracted and digested with intestinal fluid. The results showed that naked amiR-CHS1 was quickly degraded after 5 min of incubation and completely degraded after 10 min. By contrast, the amiCHS1 encapsulated by the MS2 VLP could be detected even after 10 minutes of incubation. These results indicate that the MS2 VLP could protect amiRNA from degradation in the intestinal fluid of <em>S. litura.</em></p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/result-9.jpg" alt="" data-href="" style="width: 60%;"></p><p style="text-align: justify;"><strong>Figure 7 &nbsp;</strong>Stability test of <em>S. litura</em>'s intestinal fluid against amiRNAs. Northern blotting of amiR-CHS1 in engineered <em>E. coli</em> extract after incubation with intestinal fluid (15-fold dilution) of tobacco cutworms at room temperature. Naked amiR-CHS1 is strongly degraded after 5 minutes and completely degraded after 10 minutes. In contrast, amR-CHS1 filled with VLP is relatively stable. Sample load, 5 <span style="font-family: Arial;">μg</span>.</p><p style="text-align: center;"><br></p><h3><span style="color: rgb(114, 192, 64);"><strong>Silencing </strong></span><span style="color: rgb(114, 192, 64);"><em><strong>CHS1 </strong></em></span><span style="color: rgb(114, 192, 64);"><strong>caused developmental defect of </strong></span><span style="color: rgb(114, 192, 64);"><em><strong>S. litura</strong></em></span></h3><p style="text-align: justify;">Bioassay of<em> S. litura</em> was performed till the molting stage. &nbsp;The pupation and molting of <em>S. litura </em>larvae were significantly inhibited by feeding the VLP-amiR-CHS1. </p><p style="text-align: left;"><img src="https://static.igem.wiki/teams/5044/20241001125444.png" alt="" data-href="" style=""></p><p style="text-align: justify;"><strong>Figure 8 </strong>Effects on pupation and molting of S. litura after silencing SlCHS1. (A,B) Phenotype of S. litura 25 and 32 DAF. </p><p style="text-align: justify;"><br></p><p style="text-align: justify;">Compared with CK ,VLP and amiR-CHS1, VLP-amiR-CHS1 was more effective in affecting the pupation rate and deformity rate of <em>S. litura</em>.</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/result-12-1.png" alt="" data-href="" style="width: 100%;"></p><p style="text-align: justify;"><strong>Figure 9 &nbsp;</strong>Representation (%) of developmental and mortality rates of S. litura larvae until molting stage after feeding on engineered bacteria. </p><p style="text-align: center;"><br></p><h2><span style="color: rgb(89, 191, 192);"><strong>Production of transplastomic tobacco plants</strong></span></h2><h3><span style="color: rgb(114, 192, 64);"><strong>Plastid transformation vector construction</strong></span></h3><p style="text-align: justify;">These results indicated that the expression of amiRNA in bacteria could effectively interfere with the growth and development of <em>S. litura</em>. Subsequently, <span style="color: rgb(19, 82, 0); font-size: 16px;"><strong>PLASTID </strong></span><span style="color: rgb(19, 82, 0); font-size: 16px;"> </span><span style="color: rgb(255, 255, 255); background-color: rgb(207, 19, 34); font-size: 16px;">PESTI</span><span style="color: rgb(255, 255, 255); background-color: rgb(212, 56, 13); font-size: 16px;">CIDES</span><span style="color: rgb(44, 44, 54); font-size: 16px;"><em>™</em></span> constructed plastid transformation vectors pMJ11 and pMJ12 expressing amiRNAs.</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/20240930000333.png" alt="微信图片_20240930000333.png" data-href="" style=""></p><p style="text-align: justify;"><strong>Figure 10 &nbsp;</strong>Tobacco plastid transformation vector. The selectable marker gene <em>aadA</em> is driven by the <em>psbA</em> promoter (<em>Cr</em>PpsbA) and the 3'UTR of the rbcL gene (CrTrbcL) of <em>Chlamydomonas reinhardtii</em>. G<em>accD</em>: tobacco accD promoter; Prrn: tobacco rRNA operon promoter; TrrnB: <em>rrnB</em> terminator of <em>E. coli</em>.</p><p style="text-align: center;"><br></p><h3><span style="color: rgb(114, 192, 64);"><strong>Biolistic transformation</strong></span></h3><p style="text-align: justify;">Sterile tobacco leaves were bombarded with plastid plasmids using a 1100 psi rupture disk at target distance of 9 cm. Following the biolistic bombardment, the leaf samples were diced into 5 × 5 mm and were placed on regeneration media including 500 mg/L spectinomycin.</p><p style="text-align: justify;">Authentic transplastomic clones carrying an<em> aadA</em> gene are resistant to both spectinomycin and streptomycin, whereas spontaneous spectinomycin-resistant mutants are resistant only to spectinomycin. While resistance is manifested as formation of green calli with regenerating shoots, sensitivity is indicated by formation of scanty white callus in the leaves. Resistance to streptomycin and spectinomycin indicates the presence of selectable aadA gene. However, double selection delays shoot formation. Hence, we scored the presence of <em>aadA </em>gene by resistance to streptomycin plus spectinomycin, but screen homoplastic transplastomic plants on spectinomycin.</p><p style="text-align: justify;"><strong>Table 1</strong>. Summary of plastid transformation experiments conducted in tobacco.</p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/1352.png" alt="" data-href="" style=""></p><p style="text-align: center;"><br></p><p style="text-align: center;"><img src="https://static.igem.wiki/teams/5044/20241001123329.png" alt="" data-href="" style=""></p><p style="text-align: justify;"><strong>Figure 11 &nbsp;</strong>Screening of leaves transformed with dual antibiotics. (A) Three groups of tobacco leaves transformed with the vector Nt-amiRCHS1 (B) Four groups of tobacco leaves transformed with the vector Nt-VLP-amiRCHS1.</p><p style="text-align: center;"><br></p><h3><span style="color: rgb(114, 192, 64);"><strong>Homoplasy confirmation of transplastomic plants by Southern blot</strong></span></h3><p><img src="https://static.igem.wiki/teams/5044/1210.png" alt="" data-href="" style=""></p><p style="text-align: left;"><img src="https://static.igem.wiki/teams/5044/1211.png" alt="" data-href="" style=""></p><p style="text-align: justify;"><strong>Figure 12 &nbsp;</strong>Generation of transplastic tobacco plants expressing MS2 VLPs for delivery of amiRCHS1 against <em>S. litura</em>. (A) Physical map of the targeted region in the tobacco plastid genome (ptDNA) and maps of the transformed region of the tobacco plastid genome in the Nt-VLP, Nt-amiRCHS1, and Nt-VLP-amiRCHS1 transplastic lines. The SphI restriction sites used for restriction fragment length polymorphism (RFLP) analysis of transplastomic lines are indicated, and the sizes of the restriction fragments detected in Southern blot analyses are given. (B) Southern blot analysis of plastid transformation of tobacco line. SphI-digested DNA samples were separated by electrophoresis in a 1% agarose gel and hybridized with the DIG-labeled probes shown in (A). The absence of the 4.9 kb hybridization signal of the wild-type tobacco plastid genome indicated that all transplastic lines reached homoplasmy.</p><p style="text-align: justify;"><img src="https://static.igem.wiki/teams/5044/1234.png" alt="" data-href="" style=""></p><p style="text-align: justify;"><strong>Figure 13 &nbsp;</strong>Stable inheritance of plastid transgenes and wild-type-like phenotypes of transplastomic tobacco plants. (A) Seed assays to confirm homoplasmy of transplastomic tobacco plants. Seeds obtained from wild-type (Nt-wt) and transplastomic plants were germinated on synthetic medium containing spectinomycin (500 mg L<sup>−1</sup>). Resistance of seedlings to the antibiotic and lack of segregation confirm the homoplasmic state of the transplastomic lines. Phenotypes of transplastomic tobacco lines grown in soil for 3 (B) and 6 (C) weeks. Scale bars: 1 cm in (A) and 10 cm in (B,C).</p><p style="text-align: left;"><br></p><h2><span style="color: rgb(89, 191, 192);"><strong>Reference</strong></span></h2><p>[1] Li S, Kim DS, Zhang J. Plastid-mediated RNA interference: A potential strategy for efficient pest control. Plant Cell Environ. 2023 46:2595-2605.</p><p>[2] Zhang, S., An, S., Li, Z., Wu, F., Yang, Q., Liu, Y. et al. (2015) Identification and validation of reference genes for normalization of gene expression analysis using qRT‐PCR in <em>Helicoverpa armigera </em>(Lepidoptera: Noctuidae). Gene, 555, 393–402.</p><p>[3] Wu M, Dong Y, Zhang Q, et al. (2022) Efficient control of western flower thrips by plastid‐mediated RNA interference. Proceedings of the National Academy of Sciences, 119, e2120081119.</p><p>[4] Rana S., Rajurkar, A.B., Kumar, K.K. &amp; Mohankumar, S. (2020) Comparative analysis of chitin synthaseA dsRNA mediated RNA interference for management of crop pests of different families of lepidoptera. Frontiers in Plant Science, 11, 427.</p><p>[5] Kim, D.S. &amp; Zhang, J. (2023) Strategies to improve the efficiency of RNAi‐ mediated crop protection for pest control. Entomologia Generalis, 43, 5–19.</p><p>[6] Singh, I.K., Singh, S., Mogilicherla, K., Shukla, J.N. &amp; Palli, S.R. (2017) Comparative analysis of double‐stranded RNA degradation and processing in insects. Scientific Reports, 7, 17059.</p><p>[7] Bao, W., Cao, B., Zhang, Y. and Wuriyanghan, H. (2016) Silencing of Mythimna separata chitinase genes via oral delivery of in planta-expressed RNAi effectors from a recombinant plant virus. Biotechnol. Lett. 38, 1961–1966.</p><p>[8] Diao, Pengfei, Zhang, Qimeng, Sun, Hongyu, Ma, Wenjie, Cao, Aiping, Yu, Ruonan, Wang, Jiaojiao et al. (2019) miR403a and SA are involved in NbAGO2 mediated antiviral defenses against TMV infection in Nicotiana benthamiana. Genes 10, 526–542.</p><p>[9] Huan, L., Tianming, L., Dongming, F., Furong, G., Hong, L.W., Wei, G., Xiao, F.C. et al. (2019) Chromosome level draft genomes of the fall armyworm, <em>Spodoptera frugiperda </em>(Lepidoptera: Noctuidae), an alien invasive pest in China. BioRxiv. 671560–671580.</p><p>[10] Liu, J., Smagghe, G. and Swevers, L. (2013) Transcriptional response of BmToll9-1 and RNAi machinery genes to exogenous dsRNA in the midgut of Bombyx mori. J. Insect Physiol. 59, 646–654.</p><p>[11] Chen, S., Hofius, D., Sonnewald, U. &amp; Börnke, F. (2003) Temporal and spatial control of gene silencing in transgenic plants by inducible expression of double‐stranded RNA. Plant Journal, 36, 731–740.</p><p>[12] Kumar, M., Gupta, G.P. &amp; Rajam, M.V. (2009) Silencing of acetylcholinesterase gene of <em>Helicoverpa armigera</em> by siRNA affects larval growth and its life cycle. Journal of Insect Physiology, 55, 273–278.</p><p>[13] Zhang, Q., Dong, Y. &amp; Zhang, J. (2023) Host‐induced silencing of MpPar6 confers Myzus persicae resistance in transgenic rape plants. Journal of Integrative Agriculture.</p><p>[14] Chen, Q., Shen, P., Bock, R., Li, S. &amp; Zhang, J. (2022) Comprehensive analysis of plastid geneexpression during fruit development and ripening of kiwifruit. Plant Cell Reports, 41, 1103–1114.</p><p>[15] Baum, J.A., Bogaert, T., Clinton, W., Heck, G.R., Feldmann, P., Ilagan, O., Johnson, S. et al. (2007) Control of coleopteran insect pests through RNA interference. Nat. Biotechnol. 25, 1322–1326.</p><p>[16] Eamens, A.L., Smith, N.A., Curtin, S.J., Wang, M.B. and Waterhouse, P.M. (2009) The Arabidopsis thaliana double-stranded RNA binding protein DRB1 directs guide strand selection from microRNA duplexes. RNA, 15, 2219–2235.</p><p>[17] Schwab, R., Ossowski, S., Riester, M., Warthmann, N. and Weigel, D. (2006) Highly specific gene silencing by artificial microRNAs in Arabidopsis. Plant Cell, 18, 1121–1133.</p><p>[18] Terenius, O., Papanicolaou, A., Garbutt, J.S., Eleftherianos, I., Huvenne, H., Kanginakudru, S., Albrechtsen, M. et al. (2011) RNA interference in Lepidoptera: an overview of successful and unsuccessful studies and implications for experimental design. J. Insect. Physiol. 57, 231–245.</p></section>
    <div id="entry-wrapper" class="d-none d-md-block">
      <div id="entry"><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Prediction of Effective Fragments of CHS1</div><div class="content"><p style="text-align: start;"><br></p><ul><li style="text-align: start;"><strong>Sequence Analysis</strong>: The amino acid sequences of <em>SlCHS1</em> and <em>SlCHS2</em> (chitin synthase 1 and 2 from <em>Spodoptera litura</em>) were analyzed using DASTAR software. The open reading frames (ORFs) were predicted using the ORF Finder tool, while molecular weight (MW) and isoelectric point (pI) were calculated with ExPASy. Signal peptides were predicted by SignalP 4.1, and functional domains were identified using SMART. Membrane-spanning regions were predicted with TMHMM Server v. 2.0.</li><li style="text-align: start;"><strong>Phylogenetic Analysis</strong>: A phylogenetic tree was constructed using MEGA 7.0, based on the neighbor-joining method with 1000 bootstrap resamplings, including sequences from different insect species obtained from GenBank.</li></ul></div></div><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Plasmid Construction</div><div class="content"><ul><li style="text-align: start;"><strong>Vectors Design</strong>: E. coli expression vectors were designed based on pET28a. These included:</li><li style="text-align: start;"><strong>Transformation</strong>: The constructed vectors were transferred into <em>E. coli</em> B2 (BL21ΔRNC).</li></ul></div></div><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Detection of RNA Expression</div><div class="content"><p style="text-align: start;"><br></p><ul><li style="text-align: start;"><strong>Northern Blotting</strong>: RNA expression was detected using formaldehyde denaturing gel and Northern blot. The results showed that amiRNA expression was present in strains expressing VLP, amiR-CHS1, and VLP-amiR-CHS1 after IPTG induction.</li></ul></div></div><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Detection of Protein Expression</div><div class="content"><p style="text-align: start;"><br></p><ul><li style="text-align: start;"><strong>SDS-PAGE</strong>: Protein expression was assessed using SDS-PAGE. The expression of 2×CP (28 kDa) was confirmed in strains expressing VLP and VLP-amiR-CHS1, indicating efficient expression of both protein and amiRNA.</li></ul></div></div><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Insect Feeding and Bioassays</div><div class="content"><p style="text-align: start;"><br></p><ul><li style="text-align: start;"><strong>Growth Assay</strong>: Hatched larvae of <em>S. litura</em> were fed with BL21ΔRNC strains containing different vectors. Larvae were weighed at specific time points, and growth status was recorded. Mortality rates were also monitored.</li><li style="text-align: start;"><strong>Relative Expression of SlCHS1</strong>: qPCR was used to detect the relative expression of the target gene <em>SlCHS1</em> after feeding with different BL21ΔRNC strains.</li></ul></div></div><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Stability of amiRNA</div><div class="content"><p style="text-align: start;"><br></p><ul><li style="text-align: start;"><strong>Degradation Test</strong>: To assess the stability of amiRNA, RNA from <em>E. coli</em> B2 expressing pMJ5 plasmid was incubated with <em>S. litura</em> intestinal fluid. Naked amiRNA was significantly degraded after 5 minutes, whereas amiRNA encapsulated by MS2 CP remained stable even after 10 minutes, indicating protection against RNase degradation.</li></ul></div></div><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Pupal Weight and Development</div><div class="content"><p style="text-align: start;"><br></p><ul><li style="text-align: start;"><strong>Pupation Assay</strong>: Sixth-instar larvae of <em>S. litura</em> were placed on sandy soil, and pupal development was monitored. Pupa weights were measured, and deformity and mortality rates were counted. Malformation rates were also observed during the moth stage.</li></ul></div></div><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Production of Transplastomic Tobacco Plants</div><div class="content"><p style="text-align: start;"><br></p><ul><li style="text-align: start;"><strong>Vector Construction</strong>: Based on the successful interference with <em>S. litura</em> growth, PLASTID PESTICIDES™ constructed plastid transformation vectors pMJ11 and pMJ12 to express amiRNAs in tobacco plants.</li><li style="text-align: start;"><strong>Biolistic Transformation</strong>: Tobacco leaves were transformed using biolistic methods, and transformants were selected on a medium containing double antibiotics.</li><li style="text-align: start;"><strong>Homoplasy Confirmation</strong>: Southern blot analysis was used to confirm the homoplasmy of transplastomic plants. Resistant buds were then transferred to rooting medium to induce rooting, resulting in dsRNA-producing transplastomic tobacco plants.</li></ul></div></div><div class="item"><div class="title" style="color: rgb(0, 0, 0);">Figures</div><div class="content"><p style="text-align: start;"><br></p><ul><li style="text-align: start;"><strong>Figure 1</strong>: Multiple sequence alignment of the conserved catalytic domain of chitin synthases from three insect species, highlighting characteristic motifs.</li><li style="text-align: start;"><strong>Figure 2</strong>: Schematic representation of the constructed vectors: amiR-CHS1, VLP, and VLP-amiR-CHS1.</li><li style="text-align: start;"><strong>Figure 3</strong>: Northern blot showing amiRNA accumulation in <em>E. coli</em> B2 strains.</li><li style="text-align: start;"><strong>Figure 4</strong>: SDS-PAGE gel demonstrating the expression of 2×CP in <em>E. coli</em> B2 strains.</li><li style="text-align: start;"><strong>Figure 5</strong>: Growth curve of <em>S. litura</em> after 7 days of feeding with engineered <em>E. coli</em>.</li><li style="text-align: start;"><strong>Figure 6</strong>: Relative expression of <em>SlCHS1</em> in <em>S. litura</em> after 7 days of feeding.</li><li style="text-align: start;"><strong>Figure 7</strong>: Northern blot showing the stability of amiRNA in <em>S. litura</em> intestinal fluid.</li><li style="text-align: start;"><strong>Figure 8</strong>: Pupal deformity of <em>S. litura</em> on the 25th day.</li><li style="text-align: start;"><strong>Figure 9</strong>: Feathering malformation in <em>S. litura</em> on the 32nd day.</li><li style="text-align: start;"><strong>Figure 10</strong>: Schematic of the tobacco plastid transformation vector expressing amiRNA.</li><li style="text-align: start;"><strong>Figure 11</strong>: Screening of biolistically transformed tobacco leaves on double antibiotics.</li><li style="text-align: start;"><strong>Figure 12</strong>: Southern blot confirming the homoplasmy of transplastomic tobacco plants.</li><li style="text-align: start;"><strong>Figure 13</strong>: Rooted transplastomic tobacco plants producing dsRNA.</li></ul></div></div></div>
    </div>
  </div>
  <script src="{{ url_for('static', filename = 'js/sidebar.js') }}"></script>
  <script>
    initAnchors(100, 560);
    initEntry();
  </script>
{% endblock %}
